products/telegraf.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Telegraf
+category: server-app
+versionCommand: telegraf --version
+changelogTemplate: https://github.com/influxdata/telegraf/releases/tag/v__LATEST__
+auto:
+  - git: https://github.com/influxdata/telegraf
+releases:
+  - releaseCycle: "1.24"
+    eol: false
+    releaseDate: 2022-09-12
+    latest: "1.24.2"
+    latestReleaseDate: 2022-10-03
+    link: https://github.com/influxdata/telegraf/releases/tag/v1.24.2
+  - releaseCycle: "1.23"
+    eol: false
+    releaseDate: 2022-06-13
+    latest: "1.23.4"
+    latestReleaseDate: 2022-08-16
+    link: https://github.com/influxdata/telegraf/releases/tag/v1.23.4
+  - releaseCycle: "1.22"
+    eol: true
+    releaseDate: 2022-03-23
+    latest: "1.22.4"
+    latestReleaseDate: 2022-05-17
+    link: https://github.com/influxdata/telegraf/releases/tag/v1.22.4    
+iconSlug: NA
+permalink: /telegraf
+releasePolicyLink: https://www.influxdata.com/legal/support-policy/
+releaseColumn: true
+releaseDateColumn: true
+discontinuedColumn: false
+
+---
+
+> [Telegraf](https://github.com/influxdata/telegraf) is an agent for collecting, processing, aggregating, and writing metrics.
+
+Based on a plugin system to enable developers in the community to easily add support for additional metric collection.
+InfluxData will provide Support Services for the current Minor Release (x.Y) of the Software and the immediately preceding Minor Release (x.Y)
+of the Software, inclusive of all Maintenance Releases (x.y.Z) included in the respective Minor Release. (By way of example only,
+InfluxData releases 1.8 (Major release version 1, Minor release version 8) and several maintenance releases thereafter (1.8.1, 1.8.2 and so forth).
+InfluxData then releases a new Minor Release, 1.9 (Major release version 1, Minor release version 9) and several maintenance releases
+thereafter (1.9.1, 1.9.2 and so forth). In this example, InfluxData would provide Support Services for 1.8 and 1.9 and all of its respective
+maintenance releases).
```

Comment:
```suggestion
Only the latest two minor releases are supported with maintenance patch releases. 
```

Might be worth defining what "Support Services" means. A simple policy like this one doesn't need an explanation with examples.